internal/api/attestationconfigapi/cli/parse_report.go

```diff
@@ -0,0 +1,76 @@
+/*
+Copyright (c) Edgeless Systems GmbH
+
+SPDX-License-Identifier: AGPL-3.0-only
+*/
+
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"io"
+	"strconv"
+	"strings"
+
+	"github.com/edgelesssys/constellation/v2/internal/api/attestationconfigapi"
+)
+
+// ParseSNPReport parses the SNP report and returns the version information.
+func ParseSNPReport(reader io.Reader) (attestationconfigapi.AzureSEVSNPVersion, error) {
```

Comment:
We have no stability guarantees of Constellation's cli output format. Therefore, this way of parsing is very brittle.

My suggestion is that we should write the full json (before parsing) to disk on `constellation verify` and then use the exact same parsing here.

Reply:
Is this a sensible default to write it to file at all times? or should we add a new flag to show the output as json? I can do that but was not sure this was worth the implementation effort since we also control the constellation verify output

Reply:
When we use --raw to parse the content the instance info containing the report is base64 encoded this makes the file parsing yet more obscure in the e2e test setup. Not sure we should go this route. @derpsteb what do you think?
I don't see the verify output as very unstable.
The current parsing would only fail if we rename the affected fields in the report or move the signature field in the output. But the latter could be fixed too.